--- acl_rules.js	2020-07-24 00:05:52.401585661 -0500
+++ acl_rules.js.patched	2020-07-24 00:09:44.338271197 -0500
@@ -13,6 +13,17 @@
       }]
     },
 
+    //special role for testing purposes
+    {
+      roles: 'acl_validations-role',
+      allows: [{
+        resources: [
+          'acl_validations',
+        ],
+        permissions: '*'
+      }]
+    },
+
     //models
     /**
      * Model: accession
