--- acl_rules.js	2021-01-05 10:34:54.861666483 +0100
+++ acl_rules.js.patched	2021-01-05 10:35:05.929483682 +0100
@@ -13,6 +13,17 @@
       }]
     },
 
+    //special role for testing purposes
+    {
+      roles: 'acl_validations-role',
+      allows: [{
+        resources: [
+          'acl_validations',
+        ],
+        permissions: '*'
+      }]
+    },
+    
     // model permissions
     {
       roles: 'editor',
